src/github/client/github-client-server.test.ts

```diff
@@ -30,7 +30,7 @@ describe("GitHub Client", () => {
 		perPage: number,
 		page: number,
 		expectedInstallationTokenInHeader?: string,
-		scope: nock.Scope = gheNock
+		scope: nock.Scope = gheApiNock
```

Comment:
Any changes with `gheNock` and `gheApiNock` are mine.  We weren't handling non-api nock calls correctly in the unit test setup for what Rachelle was trying to do, which is to test the top level github instance.